client/src/app/components/CreateProject/Form.js

```diff
@@ -28,10 +28,10 @@ class Form extends React.Component {
     }
   }
 
-  onChangeInput(e) {
-    e.target.name === "project-title"
-      ? this.setState({ titleValue: e.target.value })
-      : this.setState({ descriptionValue: e.target.value });
+  onChangeInput({ target }) {
+    target.name === "project-title"
```

Comment:
Another option would have been to destructure inside the method itself with `const { name, value } = event.target`